LEAF_Request_Portal/templates/sitemap.tpl

```diff
@@ -0,0 +1,23 @@
+<link rel="stylesheet" href="../libs/css/leaf.css">
+<style>
+	.flex-container {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: center;
+		align-items: center;
+	}
```

Comment:
Move inline styles to leaf.css

Reply:
Done.

Reply:
leaf.css is generated from sass under the libs folder. Make sure you add it there and run gulp watch to generate leaf.css

Reply:
Abiola was having issues with installing gulp. The css issues will be fixed in the 1723: Update Sitemap Colors jira task.